onnxruntime/python/tools/pytorch_export_contrib_ops.py

```diff
@@ -5,15 +5,16 @@
 Support for registering ONNX Runtime's built-in contrib ops with
 PyTorch-ONNX exporter (torch.onnx.export).
 """
-
 import typing
 
+import torch._C
```

Comment:
move this to inside the `try`. It's there to have a nice error message when pytorch is not installed.

Reply:
Imported torch in try instead